### PR TITLE
Post release I/O changes of the simple glazing frame feature

### PR DESCRIPTION
**Pull request by xuanluo113**
Pull request overview
---------------------
IDD and EIO change to the simple glazing model with highly conductive frames. 
- Removed IDD regulated upper limit of simple glazing U value.
- Added the adjustment ratio of simple glazing materials in the EIO window reports.

The EIO changes are expected with test files that have simple glazing models.

This is a followup to #8653 

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [x] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [x] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by xuanluo113** [comment]
@Myoldmopar Hi Edwin, since there's gonna be a bug fix release soon, I wonder if this bug-fixing branch could also get reviewed and merged into the bug fix release? Thanks!

File: idd/Energy+.idd.in [hunk 1]
```diff
@@ -5129,10 +5129,8 @@ WindowMaterial:SimpleGlazingSystem,
   N1 , \field U-Factor
        \required-field
        \note Enter U-Factor including film coefficients
-       \note Note that the effective upper limit for U-factor is 5.8 W/m2-K
```
Comment (Myoldmopar) [inline idd/Energy+.idd.in:5132]: :+1: I know this was discussed in the other PR; this is a good change.

File: src/EnergyPlus/WindowManager.cc [hunk 2]
```diff
@@ -7365,14 +7365,9 @@ namespace WindowManager {
             //                                      Write Descriptions
             print(state.files.eio,
                   "{}\n",
-                  "! <WindowConstruction>,Construction Name,Index,#Layers,Roughness,Conductance {W/m2-K},SHGC,"
+                  "! <WindowConstruction>,Construction Name,Index,#Layers,Roughness,Conductance {W/m2-K},Conductance (Before Adjusted) {W/m2-K},"
+                  "Convection Coefficient Adjustment Ratio,SHGC,"
```
Comment (Myoldmopar) [inline src/EnergyPlus/WindowManager.cc:7369]: Added new fields to the EIO, again just held off for I/O freeze.  :+1:

File: src/EnergyPlus/WindowManager.cc [hunk 3]
```diff
@@ -7539,22 +7534,11 @@ namespace WindowManager {
                               state.dataConstruction->Construct(ThisNum).TotLayers,
                               Roughness(static_cast<int>(state.dataConstruction->Construct(ThisNum).OutsideRoughness)),
                               NominalConductanceWinter,
+                              state.dataHeatBal->NominalUBeforeAdjusted(ThisNum),
+                              state.dataHeatBal->CoeffAdjRatio(ThisNum),
```
Comment (Myoldmopar) [inline src/EnergyPlus/WindowManager.cc:7538]: And the 2 new fields are added here :+1:

**Review by Myoldmopar — APPROVED** [review]
All good here!

**Comment by Myoldmopar** [comment]
@xuanluo113 the bug fix release was a minimal release purely to address issues with the space work.  It was released back on like October 16.  This is good to go into develop though.  Assuming a local build (almost done) is clean, then I'll be merging this in now with the expected EIO diffs in place.

**Comment by Myoldmopar** [comment]
Yep, all good, merging.  Thanks @xuanluo113

**Comment by mjwitte** [comment]
This needs a description of the eio changes in OuputChanges9-6-0-.....md.

**Comment by xuanluo113** [comment]
@mjwitte Please check out #9179 for the added descriptions.

**Pull request merged**